test/README.md

```diff
@@ -0,0 +1,18 @@
+# koa-body Test Info
+
+Running tests:
+```sh
+npm install
+npm test
+```
+
+## Coverage
+To run coverage, you must be using [c8's supported version](https://github.com/bcoe/c8#supported-nodejs-versions) of Node.js, v10.12.0+.
+
+To get an HTML coverage report, run the following commands:
+```sh
+npm run coverage
+./node_modules/.bin/c8 report --reporter=html
```

Comment:
Could add this to package.json scripts - I feel like that's pretty harmless.
```
scripts: {
   ...
   "coverage:html" : "npm run coverage && c8 report --reporter=html && npx http-server coverage"
}
```

Reply:
IMO that's not a great idea; I have some personal bias against npx, being very loose-goose in what it does and very much against the concepts of repeatable, deterministic results.  If we're going to depend on something, add it as a devdependency.

Reply:
Not a great idea to add html coverage reports as a script, or just the usage of `npx`? 

I don't care so much about how the results are opened. Agreed I would use a devDependency as well, but even just `... && echo "Report generated, open coverage/index.html in your browser"` would suffice just fine I'd think

Reply:
I think the echo message is a happy medium for the first pass at this PR. If we find we're doing a lot of manual opening of the report, let's change it to automatically open or use a http server.

Good suggestions!

Reply:
Specifically about using npx to launch an http-server. If it's effectively a devdependency, make it one.